app/views/drawings/_form.html.haml

```diff
@@ -1,53 +1,49 @@
-.container
-  %h3
+.col-xs-12.text-center
+  %h1.add-margin.top.bottom
     = form_title
-  .row.drawings-wrapper
-    = simple_form_for @drawing, html: { multipart: true } do |f|
-      .col-md-5.col-sm-12.col-xs-12
+.row.drawings-wrapper.no-padding
+  .col-xs-12.col-sm-11.position.central.no-padding
+    = simple_form_for @drawing, html: {multipart: true } do |f|
+      .col-xs-12.col-md-4
         .image-wrap.text-center
-          %h4 Upload an image
-          %i (this is required)
           = image_tag preview_image(@drawing), id: 'image-preview', class: 'img-responsive'
-          %br
```

Comment:
Is the change to remove this space between the image placeholder (or image) and "Allowed image type" text intentional?